src/Publisher/Builder/ManualDocBuilder.js

```diff
@@ -56,15 +56,15 @@ export default class ManualDocBuilder extends DocBuilder {
   _getManualConfig() {
     const m = this._config.manual;
     const manualConfig = [];
-    if (m.overview) manualConfig.push({label: 'Overview', paths: m.overview});
-    if (m.installation) manualConfig.push({label: 'Installation', paths: m.installation});
-    if (m.tutorial) manualConfig.push({label: 'Tutorial', paths: m.tutorial});
-    if (m.usage) manualConfig.push({label: 'Usage', paths: m.usage});
-    if (m.configuration) manualConfig.push({label: 'Configuration', paths: m.configuration});
-    if (m.example) manualConfig.push({label: 'Example', paths: m.example});
+
+    for (var file in m) {
+      if (file != 'asset') {
```

Comment:
I would suggest 2 edits here:

1. Use `!==` for faster and more accurate comparisons.
2. I would suggest excluding `reference` just in case, so that you don't end up with multiple `Reference` entries.

```js
if (file !== 'asset' && file !== 'reference') {
```